From 7a26ce92adcdcbd8eca96b8d51c7cf91db01d614 Mon Sep 17 00:00:00 2001
From: Cyan <cyan@cyano.uk>
Date: Fri, 15 Nov 2024 16:49:55 +0800
Subject: [PATCH 41/47] templates: make UEFI fwsetup menu entry label
 translatable

When the "UEFI Firmware Settings" entries was added in commit 46d76f8fe
"templates: Output a menu entry for firmware setup on UEFI FastBoot
systems," it was not marked with Gettext identifiers. Mark this
string for localization.

Signed-off-by: Xinhui Yang <cyan@cyano.uk>
---
 util/grub.d/30_uefi-firmware.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/util/grub.d/30_uefi-firmware.in b/util/grub.d/30_uefi-firmware.in
index 1c2365ddb..9cbeac6cc 100644
--- a/util/grub.d/30_uefi-firmware.in
+++ b/util/grub.d/30_uefi-firmware.in
@@ -26,7 +26,7 @@ export TEXTDOMAINDIR="@localedir@"
 
 . "$pkgdatadir/grub-mkconfig_lib"
 
-LABEL="UEFI Firmware Settings"
+LABEL=$(gettext_printf "UEFI Firmware Settings")
 
 gettext_printf "Adding boot menu entry for UEFI Firmware Settings ...\n" >&2
 
-- 
2.48.1

